Film advertising

MikaelaMathisson opened the pull request:
Added the boxes of film advertising to index.html and news.html.

MikaelaMathisson committed:
Adding iframe with trailer video youtube

MikaelaMathisson committed:
Centering iframe, adding title and premier date

MikaelaMathisson committed:
Adding filmadvertising to news.html

sebastianpiresmolin commented on index.html:
```diff
@@ -19,7 +19,19 @@
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowfullscreen
+      ></iframe>
+      <h2>Hammarsköljd</h2>
```
Hammarskjöld*

MikaelaMathisson replied:
Nice catch! Will change this when I can

sebastianpiresmolin commented on index.html:
```diff
@@ -19,7 +19,19 @@
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowfullscreen
+      ></iframe>
+      <h2>Hammarsköljd</h2>
```
Maybe add transform(y) in css so that the text ''overlays'' the iframe? I think that would look nice!

MikaelaMathisson replied:
Good idea! I Will try to do that. If I have any problems to do it, I would love to get your help. 👍🏽 I will let you know

MikaelaMathisson committed:
Typo fix

MikaelaMathisson committed:
Adding translateY to move the titles over the iframe

MikaelaMathisson committed:
Merge branch 'main' into film-advert

MikaelaMathisson commented:
@sebastianpiresmolin I fixed the typo and added the translate y. 
I did the translate y to make the title of the movie go up in the iframe but i couldnt fit the premiere date without it interfering with the iframe. So i put it under. I think it looks fine. 
If you have time, please check this and test.
